python/cuml/test/test_fil.py

```diff
@@ -260,30 +262,33 @@ def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
 
     if n_classes == 2:
         assert array_equal(fil_preds, skl_preds_int)
-        fil_proba = np.asarray(fm.predict_proba(X_validation))
-        fil_proba = np.reshape(fil_proba, np.shape(skl_proba))
-        assert np.allclose(fil_proba, skl_proba, 1e-3)
+    fil_proba = np.asarray(fm.predict_proba(X_validation))
+    fil_proba = np.reshape(fil_proba, np.shape(skl_proba))
+    proba_threshold = 1e-3 if n_classes == 2 else 0.1
+    assert np.allclose(fil_proba, skl_proba, proba_threshold)
```

Comment:
preferable to use assert_allclose from np.testing as it gives nicer error messages